schema/manifest_backwards_compatibility_test.go

```diff
@@ -136,14 +136,14 @@ func TestBackwardsCompatibilityManifest(t *testing.T) {
 		//
 		// curl -L -H "Authorization: Bearer ..." -H \
 		// "Accept: application/vnd.docker.distribution.manifest.v2+json" \
-		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:888206c77cd2811ec47e752ba291e5b7734e3ef137dfd222daadaca39a9f17bc
+		// https://registry-1.docker.io/v2/library/docker/manifests/sha256:d901d5244c94db8151415bf3a1d3a0b10e7468e4997786b01c287cbcbaeca446
```

Comment:
> However, when the modified mediatype, we must then modify the digest. This will resolve the warning

To satisfy the spirit of these compat tests, you cannot modify the source content (or its digest) at all.  The point is that these examples are caught-in-the-wild blobs, and that after flushing that wild-caught content through `convertFormats` you get a valid OCI blob.  The compatibility claim will be less convincing if it becomes “we edit the wild blob by hand and then flush it through `convertFormats` so we're compatible” ;).

And you *do* get a valid OCI blob (because we only [SHOULD `application/vnd.oci.image.config.v1+json` for a manifest's `config` descriptor][1]).  The “has an unknown media type” [error you get][2] is because the current tests are stricter than the spec and provide no way to [select between “MUST all compliant handlers support this blob?” and “is this blob valid?”][3].

[1]: https://github.com/opencontainers/image-spec/blame/v1.0.0-rc3/manifest.md#L40
[2]: https://github.com/opencontainers/image-spec/pull/510#issue-197963184
[3]: https://github.com/opencontainers/image-spec/pull/510#discussion_r94497050